vm/src/obj/pystr.rs

```diff
@@ -264,4 +265,41 @@ pub trait PyCommonString<E> {
     fn py_rjust(&self, width: usize, fillchar: E) -> Self::Container {
         self.py_pad(width - self.chars_len(), 0, fillchar)
     }
+
+    fn py_removeprefix<FC>(
+        &self,
+        prefix: &Self::Container,
+        prefix_len: usize,
+        is_prefix: FC,
+    ) -> &Self
+    where
+        FC: Fn(&Self, &Self::Container) -> bool,
+    {
+        //if self.py_starts_with(prefix) {
+        if is_prefix(&self, &prefix) {
+            return self.get_bytes(Range {
+                start: prefix_len,
+                end: self.bytes_len(),
+            });
+        }
+        &self
```

Comment:
I actually want to suggest remove `return`.

```suggestion
        if is_prefix(&self, &prefix) {
            self.get_bytes(prefix_len..self.bytes_len())
        } else {
            &self
        }
```